src/include/miopen/backend_api.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <miopen/miopen.h>
+#include <miopen/tensor.hpp>
+#include <miopen/backend_api_builders.hpp>
```

Comment:
rename this file to something like `backend_tensor.hpp` and include this header file `backend_api.hpp` in `backend_tensor.hpp`. This to me is the more logical `include order`. 
Then in `backend_api.cpp` you can include both the header files.

Reply:
agree